manifests/config.pp

```diff
@@ -3,10 +3,16 @@
 class selinux::config {
   assert_private()
 
-  selinux_state { 'set_selinux_state': ensure => $::selinux::ensure }
+  selinux_state { 'set_selinux_state':
```

Comment:
Looks like this was wrong originally.

A reboot is required if you go from:
* off -> enforcing
* enforcing -> off

A reboot is *not* required if:
* enforcing -> permissive
* permissive -> enforcing

Reply:
I think I will have to notify this from the provider then. I will wrap it into SIMP-3758

Reply:
@jeefberkey You can just use the `fact` to get the current state and compare against that.